tests/deployment/test_client_server.py

```diff
@@ -106,23 +107,18 @@ def test_client_server_sklearn(
     with pytest.raises(AttributeError, match=".* model is not compiled.*"):
         client_server_simulation(x_train, x_test, model, default_configuration)
 
-    fhe_circuit = model.compile(
-        x_train, default_configuration, **extra_params, show_mlir=(n_bits <= 8)
-    )
-
-    if get_model_name(model) == "KNeighborsClassifier":
-        # Fit the model
-        with warnings.catch_warnings():
-            # Sometimes, we miss convergence, which is not a problem for our test
-            warnings.simplefilter("ignore", category=ConvergenceWarning)
-            model.fit(x, y)
+    fhe_circuit = model.compile(x_train, default_configuration, **extra_params, show_mlir=False)
 
     max_bit_width = fhe_circuit.graph.maximum_integer_bit_width()
     print(f"Max width {max_bit_width}")
 
     # Check that the FHE execution is correct.
     # With a global_p_error of 1/100_000 we only allow one run.
+
+    # Check the real FHE prediction vs the clear ones
+    # The simulated predictions are not considered in this test
     check_is_good_execution_for_cml_vs_circuit(x_test, model, simulate=False, n_allowed_runs=1)
+    # Check client/server FHE predictions vs the FHE predictions of the dev model
```

Comment:
could you add add a space above this comment for better readability ?

Reply:
bad rebase !
Thanks

Reply:
(I think you forgot to add it)

Reply:
> bad rebase !
> Thanks

My bad, it was meant for the above comment lol not for this one.
Yes I'll add a space.